src/renderer/helpers/apiHelper.ts

```diff
@@ -0,0 +1,11 @@
+import { parse } from 'url'
```

Comment:
hm. do we really need this package ? seems [URL class](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL) should be enough ?